src/common/gui/SurgeGUIEditor.cpp

```diff
@@ -5828,8 +5830,19 @@ VSTGUI::CCommandMenuItem* SurgeGUIEditor::addCallbackMenu(VSTGUI::COptionMenu* t
 #if TARGET_VST3
 Steinberg::tresult PLUGIN_API SurgeGUIEditor::onSize(Steinberg::ViewRect* newSize)
 {
-   float izfx = newSize->getWidth() * 1.0 / getWindowSizeX() * 100.0;
-   float izfy = newSize->getHeight() * 1.0 / getWindowSizeY() * 100.0;
+   Steinberg::ViewRect currentSize = *newSize;
```

Comment:
So implicit here is that currentSize.top == 0 and currentSize.left == 0. I am sure they are but do we want to check somehow below (since you move the bottom and assume that is changing the width).

Reply:
Didn't seem like those values were used at all.

Reply:
They are used below when you can currentSize->getWidth() which is right - left.